tests/data/testReadTextCatalog_nullValues.csv

```diff
@@ -0,0 +1,3 @@
+name, ra, dec, counts, flux, resolved
+£object1, -5.0, 10.0, 1000, 1.1, 1
```

Comment:
Do you want the pound-symbol at the start of this?

Reply:
Not needed. This was copied from the `testReadTextCatalog.csv`, but I'm not checking non-ASCII characters here, so I'll remove it.